packages/core/src/client.ts

```diff
@@ -2267,11 +2268,15 @@ export class MedplumClient extends EventTarget {
     const retryDelay = 200;
     let response: Response | undefined = undefined;
     for (let retry = 0; retry < maxRetries; retry++) {
-      response = (await this.fetch(url, options)) as Response;
-      if (response.status < 500) {
-        return response;
+      try {
+        response = (await this.fetch(url, options)) as Response;
+        if (response.status < 500) {
```

Comment:
Unless if it's a 500 error, the response will still return, thus the only way for an error message is a network error, which is important because every browser had a different Error message and was difficult to key in the right conditional if we wanted to be more specific